src/evagram_input/input_tool/input_tool.py

```diff
@@ -26,12 +29,22 @@ def input_data(self):
             self.experiment_id = self._add_current_experiment(self.experiment, self.owner_id)
             self._run_task()
         except psycopg2.OperationalError as err:
-            print(err)
+            raise
+        except FileNotFoundError as err:
+            raise
+        except RuntimeError as err:
+            raise
         else:
+            print((f"Data successfully added into evagram for '{self.owner}' and "
+                   f"'{self.experiment}' experiment! Exiting session with: "))
+            print("Number of diagnostics added:", self._diagnostics)
+            print("Number of duplicates with same filename found:", self._duplicates)
```

Comment:
Consider forming this message inside a  `__repr__` function?  Or if not `__repr__`, then a self attribute  called "status".  Then print the status at the end.

Reply:
Sounds good, I can include the return message in the `__repr__` function that way only the instance needs to be printed to view the log.